src/backend/distributed/operations/worker_node_manager.c

```diff
@@ -499,6 +499,21 @@ DistributedTablePlacementNodeList(LOCKMODE lockMode)
 }
 
 
+/*
+ * IsSingleNodeCluster returns true if it's a single node cluster, false otherwise.
+ */
+bool
+IsSingleNodeCluster(void)
```

Comment:
this method is no longer needed

Reply:
yes, but I think it might be useful in the future. shouldn't we keep it?

Reply:
Not really, because we shouldn't have an unused method in the codebase